app/packages/viz3d/src/draco_loader.ts

```diff
@@ -0,0 +1,186 @@
+import * as three from "three"
+import * as draco from "draco3d"
+
+import * as draco_worker_lib from "./draco_worker"
+import draco_worker from "./draco_worker?worker"
+
+import * as worker_util from "./worker_util"
+export {LogLevel} from "./worker_util"
+
+//export { DracoTaskConfig } from "./draco_worker"
+
+// TODO: Improve performance.
+// As of Sun Feb 6 14:15:05 PST 2022 this can
+// load ~5mb point clouds (compressed to ~650kb) in ~65ms/each. 
+// This might be too slow for smooth UI.
+
+// TODO: Might be faster to do batch processing.
+// Measure overhead incurred from going back and 
+// forth from the worker to the client side.
+
+// TODO: Look into using some sort of shared ArrayBuffer 
```

Comment:
You cannot use shared array buffers, it is not supported by all browsers.

Please use `transfer` in [`postMessage`](https://developer.mozilla.org/en-US/docs/Web/API/Worker/postMessage) to pass ArrayBuffers between threads . This is how masks are passed. See `worker.ts`